blocks/shared-styles/_children/promo-image/styles.scss

```diff
@@ -1,3 +1,18 @@
 .promo-image {
   position: relative;
+
+  .promo-label {
+    .label {
+      color: #fff;
+      margin-left: 8px;
+      flex-grow: 0;
```

Comment:
```suggestion
```

Was there a reason this was added compared with https://github.com/WPMedia/fusion-news-theme-blocks/pull/890/files#diff-249277bd5886fedffad4be3e3aa2d4254e03759db78db5c157ef65c1037093bbL35

flex-grow defaults to 0

Reply:
thanks for updating with the previous styles

Reply:
yes i needed to use a combination of those styles, i accidentally replaced, some, all good.